Ladybird/Utilities.cpp

```diff
@@ -24,13 +25,14 @@ ErrorOr<String> application_directory()
 void platform_init()
 {
     s_serenity_resource_root = [] {
-        auto const* source_dir = getenv("SERENITY_SOURCE_DIR");
-        if (source_dir) {
-            return ByteString::formatted("{}/Base", source_dir);
+        if (auto source_dir = Core::Environment::get("SERENITY_SOURCE_DIR"sv); source_dir.has_value()) {
+            return ByteString::formatted("{}/Base", *source_dir);
         }
-        auto* home = getenv("XDG_CONFIG_HOME") ?: getenv("HOME");
-        VERIFY(home);
-        auto home_lagom = ByteString::formatted("{}/.lagom", home);
+        auto home = Core::Environment::get("XDG_CONFIG_HOME"sv);
+        if (!home.has_value())
+            home = Core::Environment::get("HOME"sv);
+        VERIFY(home.has_value());
```

Comment:
This verify is already present in `*home`, though I don't really mind having it here too.